backend/qemu.pm

```diff
@@ -1127,8 +1127,10 @@ sub handle_qmp_command ($self, $cmd, %optargs) {
         bmwqemu::fctinfo("Skipping the following qmp_command because QEMU_ONLY_EXEC is enabled:\n$line");
         return undef;
     }
-    my $wb = defined $optargs{send_fd} ? tinycv::send_with_fd($sk, $line, $optargs{send_fd}) : syswrite($sk, $line);
-    die "handle_qmp_command: syswrite failed $!" unless ($wb == length($line));
+    $SIG{__DIE__} = undef;    # prevent custom die handler so that a broken pipe can be handled
```

Comment:
Don't we need to restore that handler again after leaving this function? This could be done using https://docs.mojolicious.org/Mojo/Util#scope_guard. Maybe it would also work to use `local $SIG{__DIE__} = undef;`.

Reply:
yeah, i was also going to comment on that line